core/object.cpp

```diff
@@ -741,8 +741,11 @@ static void _test_call_error(const StringName &p_func, const Variant::CallError
 
 void Object::call_multilevel(const StringName &p_method, const Variant **p_args, int p_argcount) {
 	ZoneScoped;
-	CharString c = Profiler::stringify_method(p_method, p_args, p_argcount);
-	ZoneName(c.ptr(), c.size());
+	const String class_name = get_class();
+	const CharString call_details = Profiler::stringify_method(class_name, p_method, p_args, p_argcount);
```

Comment:
Getting a compile error
```
core\object.cpp(880): error C2660: 'Profiler::stringify_method': function does not take 4 arguments
```

is there supposed to be a change to the tracy submodule before this one goes in?

Reply:
Yes as noted in the Discord thread, I can't publish my branch on pahdolabs/godot_tracy